src/firmware/app/control/trajectory_planner_test.cpp

```diff
@@ -1288,50 +768,1631 @@ TEST_F(TrajectoryPlannerTest, velocity_trajectory_curved_path_low_acceleration)
+    path_parameters.path                = {.x = {0, 0, 0, 0}, .y = {0, 0, 0, 0}};
+    path_parameters.orientation_profile = {.coefficients = {0, 0, 0, 0}};
+
+    float max_allowable_speed_profile[TRAJECTORY_PLANNER_MAX_NUM_ELEMENTS];
+    TrajectorySegment_t segment_lengths[TRAJECTORY_PLANNER_MAX_NUM_ELEMENTS];
+
+
+    PositionTrajectory_t position_trajectory;
+    PositionTrajectoryElement_t
+        position_trajectory_elements[TRAJECTORY_PLANNER_MAX_NUM_ELEMENTS];
+
+    VelocityTrajectory_t velocity_trajectory;
+    VelocityTrajectoryElement_t
+        velocity_trajectory_elements[TRAJECTORY_PLANNER_MAX_NUM_ELEMENTS];
+
+    position_trajectory.trajectory_elements = position_trajectory_elements;
+    velocity_trajectory.trajectory_elements = velocity_trajectory_elements;
+
+    position_trajectory.path_parameters = path_parameters;
+    velocity_trajectory.path_parameters = path_parameters;
+
+    app_trajectory_planner_getMaxAllowableSpeedProfile(
+        position_trajectory.path_parameters.path,
+        position_trajectory.path_parameters.num_segments,
+        position_trajectory.path_parameters.t_start,
+        position_trajectory.path_parameters.t_end,
+        position_trajectory.path_parameters.max_allowable_linear_acceleration,
+        max_allowable_speed_profile);
+    app_trajectory_planner_generateStatesAndReturnSegmentLengths(&position_trajectory,
+                                                                 segment_lengths);
+
+    app_trajectory_planner_createForwardsContinuousLinearSpeedProfile(
+        &position_trajectory, max_allowable_speed_profile, segment_lengths);
+    app_trajectory_planner_createForwardsContinuousAngularSpeedProfile(
+        &position_trajectory, segment_lengths);
+
+    app_trajectory_planner_modifyTrajectoryToBackwardsContinuous(&position_trajectory,
+                                                                 segment_lengths);
+    app_trajectory_planner_generatePositionTrajectoryTimeProfile(&position_trajectory,
+                                                                 segment_lengths);
+
+    app_trajectory_planner_generateVelocityTrajectory(&position_trajectory,
+                                                      &velocity_trajectory);
+
+    for (unsigned int i = 0; i < velocity_trajectory.path_parameters.num_segments; i++)
+    {
+        EXPECT_FLOAT_EQ(velocity_trajectory.trajectory_elements[i].angular_velocity, 0);
+        EXPECT_FLOAT_EQ(velocity_trajectory.trajectory_elements[i].linear_velocity.x, 0);
+        EXPECT_FLOAT_EQ(velocity_trajectory.trajectory_elements[i].linear_velocity.y, 0);
+        EXPECT_FLOAT_EQ(velocity_trajectory.trajectory_elements[i].time, 0);
+    }
+}
+
+TEST_F(TrajectoryPlannerTest,
+       test_generate_velocity_trajectory_linear_path_no_orientation)
+{
+    // Create a scenario where the linear path parameters will limit the speed of the
+    // trajectory by using angular parameters that are much greater
```

Comment:
This comment seems to be copied a lot of tests. I think at least a few instances of it must be incorrect / out-of-place? Could you do a pass of all them to double check / update them? Here for example, I don't think it makes much sense.

Reply:
Good catch. I went through all of them and fixed the comments